code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -83,6 +83,30 @@
 		if("cry")
 			message = "<B>[comm_paygrade][src]</B> cries."
 
+		if("dab")
+			message = "<B>[comm_paygrade][src]</B> dabs."
+			log_message(message, LOG_EMOTE)
+
+			for(var/mob/M in dead_mob_list)
```

Comment:
Remove this copypasta till the line above the first limb var. It's already handled at the end.

Reply:
this and the early return was to have the dab message happen before the delimb, else it happens after.